metrics/prometheus/windows/prometheus.yml

```diff
@@ -8,6 +8,6 @@ scrape_configs:
     static_configs:
       - targets: ['host.docker.internal:9090'] # Only works on Docker Desktop for Windows
 
-  - job_name: 'service-collector'
+  - job_name: 'mining-node-collector'
     static_configs:
-      - targets: ['192.168.65.1:8080']
+      - targets: ['192.168.65.1:9000']
```

Comment:
the port checks out, but what about the IP? shouldn't it be something non-hardcoded?

Reply:
Iirc, `192.168.65.1` is the old "magical" docker IP address that resolves to the host IP (now `host.docker.internal`). However as I didn't have any windows machine/VM around, I just retained this IP.